packages/fxa-auth-server/docs/metrics-events.md

```diff
@@ -390,9 +390,11 @@ has the same schema as `daily_multi_device_users`.
 | `device.created`       | A device record has been created for a Sync account.         |
 | `device.updated`       | Device record is updated on a Sync account.                  |
 | `device.deleted`       | Device record has been deleted from a Sync account.          |
-| `oauth.token.created`  | An oauth access token has been issued for an account.        |
+| `oauth.token.created`  | An oauth access token has been issued for an account. **      |
```

Comment:
@6a68 Here is where I added the note about AET, I also filed https://jira.mozilla.com/browse/FXA-2295 as a follow up when we have more of the feature completed.

Reply:
This looks good as a starting point :+1: